graylog2-server/src/main/java/org/graylog2/streams/StreamRouterEngine.java

```diff
@@ -49,23 +49,13 @@
     private static final Logger LOG = LoggerFactory.getLogger(StreamRouterEngine.class);
 
     private final List<Stream> streams;
-    private final StreamFaultManager streamFaultManager;
-    private final StreamMetrics streamMetrics;
-    private final TimeLimiter timeLimiter;
-    private final long streamProcessingTimeout;
+    protected final StreamFaultManager streamFaultManager;
+    protected final StreamMetrics streamMetrics;
+    protected final TimeLimiter timeLimiter;
+    protected final long streamProcessingTimeout;
```

Comment:
Why did you change the visibility of those final fields? They're only being accessed in this very class.